robot/Cumulus/resources/NPSP.robot

```diff
@@ -386,6 +387,14 @@ Click Field And Select Date
     [Arguments]    ${field}    ${date}
     Click Element With Locator    bge.field-input    ${field}    
     Click BGE Button    ${date}    
+    
+Run Accessibility Check
```

Comment:
I love combining the individual methods into a single call so it's a one-liner to run an accessibility check.  Is it possible to have this `Run Accessibility Check` function/method defined "globally" so each robot test doesn't have to re-create it.  e.g. some setup file defines this method, and all robot tests use the common setup file.